acl/policy_test.go

```diff
@@ -494,6 +494,19 @@ func TestParse(t *testing.T) {
 			"Invalid variable policy: no variable paths in namespace dev",
 			nil,
 		},
+		{
+			`
+			namespace "dev" {
+			  variables {
+                path "/nomad/job" {
+			      capabilities = ["read", "write"]
+                }
+			  }
+			}
+			`,
+			"Invalid variable path '/nomad/job' in namespace dev: cannot start with a leading '/'",
```

Comment:
If you accept the above change. nbd
```suggestion
			"Invalid variable path \"/nomad/job\" in namespace dev: cannot start with a leading '/'",
```